src/field.h

```diff
@@ -267,8 +267,10 @@ static void secp256k1_fe_sqr(secp256k1_fe *r, const secp256k1_fe *a);
 /** Compute a square root of a field element.
  *
  * On input, a must be a valid field element with magnitude<=8; r need not be initialized.
- * Performs {r = sqrt(a)} or {r = sqrt(-a)}, whichever exists. The resulting value
- * represented by r will be a square itself. Variables r and a must not point to the same object.
+ * Performs {r = sqrt(a)} and returns 1 if sqrt(a) exists.
+ * Performs {r = sqrt(-a)} and returns 0 otherwise; then sqrt(-a) exists.
```

Comment:
nit: maybe case distinction like this is clearer?
```
 * If sqrt(a) exists, performs {r = sqrt(a)} and returns 1.
 * Otherwise, sqrt(-a) exists. The function performs {r = sqrt(-a)} and returns 0.
```

Reply:
Done.